tests/test_workers/test_pkg_managers/test_general_js.py

```diff
@@ -17,9 +17,14 @@
 @mock.patch("cachito.workers.pkg_managers.general_js.generate_and_write_npmrc_file")
 @mock.patch("cachito.workers.pkg_managers.general_js.RequestBundleDir")
 @mock.patch("cachito.workers.pkg_managers.general_js.run_cmd")
-def test_download_dependencies(mock_run_cmd, mock_rbd, mock_gawnf, mock_td):
+@mock.patch("shutil.move")
```

Comment:
Could you verify that `shutil.move` was properly called?

Reply:
I wanted to originally but when I played around a bit last evening, I couldn't figure out verifying paths as they were mock objects. The code was still reachable and so I figured it might not be a requirement.
But I played around a bit more today after your comment and realized that the existing code had `RequestBundleDir` mocked. I made necessary changes and it seems like the code now checks all calls for `shutil.move`.
Pushing the updates as part of next push.